exec/selected/selected.go

```diff
@@ -63,14 +63,18 @@ func (sf *SchemaField) ToSelectedField() *types.SelectedField {
 	if sf == nil {
 		return nil
 	}
-	return &types.SelectedField{
+	res := &types.SelectedField{
 		Name:       sf.Name,
 		TypeName:   sf.Field.Type.String(),
 		Alias:      sf.Alias,
 		Fields:     selsToSelectedFields(sf.Sels),
-		Args:       sf.Args,
+		ArgsMap:    sf.Args,
 		Directives: append(sf.Directives, sf.Field.Directives...),
 	}
+	if sf.PackedArgs.IsValid() {
+		res.Args = sf.PackedArgs.Interface()
```

Comment:
can you please elaborate on how it's supposed to be used?

Reply:
The caller of the field selection knows the argument type used in the place where the lookup is performed. Therefore they can cast it to that type and do whatever they like.